src/App.php

```diff
@@ -141,7 +144,7 @@ public function bootstrap()
                 $lti_token_params = http_build_query($_SESSION['lti_post']);
                 $activity_studio_link = CURRIKI_STUDIO_HOST . "/lti-tools/activity/$activity_id";
                 $redirect_to_studio_url = $activity_studio_link . "?" . $lti_token_params;
-                foreach(['user_id', 'tool_platform', 'is_learner', 'submission_id', 'course_id'] as $extra_param) {
+                foreach(['user_id', 'tool_platform', 'is_learner', 'submission_id', 'course_id', 'custom_course_name', 'custom_api_domain_url', 'custom_course_code'] as $extra_param) {
```

Comment:
Better to move array in foreach out to to a variable to avoid horizontal scroll.